docs/digital-wallets.md

```diff
@@ -16,36 +16,36 @@ These payment options offer a seamless and convenient way for your customers to
 
 ### Web
 
-| Browser | Scenario | Apple Pay | Google Pay |
-|:-------:|:--------:|:---------:|:----------:|
-| Chrome (and others) | Desktop |    ❌     |     ✅     |
-| Chrome (and others) | Mobile |    ❌     |     ✅     |
-| Chrome (and others) | Embed with an iframe - Desktop |    ❌     |     ✅     |
-| Chrome (and others) | Embed with an iframe - Mobile |    ❌     |     ✅     |
-| Safari | Desktop |    ✅     |     ✅     |
-| Safari | Mobile |    ✅     |     ✅     |
-| Safari | Embed with an iframe - Desktop | ✅ |     ✅     |
-| Safari | Embed with an iframe - Mobile | ✅ |     ✅     |
+| Browser | Scenario | Apple Pay | Google Pay | Pix |
```

Comment:
The changes on this file we can revert. PIX is not a digital wallet. And it is supported on every scenario like cards.

We need to improve instead the `Environments` section with sandbox testing amounts.